docs/cli/cli-spans.md

```diff
@@ -0,0 +1,71 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: CLI
+--->
+
+# Semantic Conventions for CLI (Command Line Interface) programs
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions to apply when instrumenting CLI programs.
+A CLI span is defined in terms of [process attributes][ProcessRegistry].
```

Comment:
I don't understand what this sentence means. Is it necessary?

Reply:
I'll drop it since all the attributes in the table link the the process registry already.